channels/br.m3u

```diff
@@ -101,28 +87,24 @@ https://v4-slbps-sambavideos.akamaized.net/live/3282%2C8114%2Cec4b5a296d97fa99bf
 http://streaming03.zas.media:1935/redemaoamiga/redemaoamiga/live.m3u8
 #EXTINF:-1 tvg-id="RedeRC.br" tvg-name="Rede RC" tvg-country="BR" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/nc95BPy.png" group-title="Local",Rede RC (360p)
 http://tv02.logicahost.com.br:1935/rederc/rederc/live.m3u8
-#EXTINF:-1 tvg-id="rs21.br" tvg-name="RS21" tvg-country="BR" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/ayvc4Ni.jpg" group-title="Religious",Rede Seculo 21 (360p)
-https://dhxt2zok2aqcr.cloudfront.net/live/rs21_360p614k/index.m3u8
 #EXTINF:-1 tvg-id="RedeSuper.br" tvg-name="Rede Super" tvg-country="BR" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/3ODbxcM.png" group-title="Religious",Rede Super (432p)
 https://api.new.livestream.com/accounts/10205943/events/3429501/live.m3u8
+#EXTINF:-1 tvg-id="RedeSeculo21.br" tvg-name="Rede Século 21" tvg-country="BR" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/vvrCWVo.jpg" group-title="Religious",Rede Século 21
+https://dhxt2zok2aqcr.cloudfront.net/live/rs21_360p614k/index.m3u8
 #EXTINF:-1 tvg-id="RedeSeculo21.br" tvg-name="Rede Século 21" tvg-country="BR" tvg-language="Portuguese" tvg-logo="https://i.imgur.com/vvrCWVo.jpg" group-title="Religious",Rede Século 21 (240p) [Not 24/7]
 https://d1gs793b1l6k9i.cloudfront.net/out/v1/a3aac39375df40a1a564635d7b41e90c/index.m3u8
 #EXTINF:-1 tvg-id="RedeTVES.br" tvg-name="Rede TV! ES" tvg-country="BR" tvg-language="Portuguese" tvg-logo="https://www.livetvnews.com.br/wp-content/uploads/2019/03/Logo-RedeTV.png" group-title="General",Rede TV! ES
 https://hls.brasilstream.com.br/live/redetves/redetves/tracks-v1a1/mono.m3u8
```

Comment:
redundant tracks-v1a1

Reply:
It's because the playlist file only has got this version, not useful to keep it.